tutorials/not_so_obvious_python_stuff.ipynb

```diff
@@ -404,16 +391,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
-   "metadata": {
-    "collapsed": false
-   },
+   "execution_count": 7,
+   "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "\"datetime.time(0,0,0)\" (Midnight) -> False\n",
+      "\"datetime.time(0,0,0)\" (Midnight) -> True\n",
       "\"datetime.time(1,0,0)\" (1 am) -> True\n"
      ]
```

Comment:
I was a bit confused why this would change and re-ran the code. It seems like this was changed in recent Python versions.

I.e., in 3.4, it is

```datetime.time(0,0,0)\" (Midnight) -> False```

in 3.5 and 3.6 it is 

```datetime.time(0,0,0)\" (Midnight) -> True```

Could you add a note about this to the notebook. E.g., something like

Please note that Python version <= 3.4.5 evaluated the first statement `bool(datetime.time(0,0,0))` as `False`, which was regarded counter-intuitive, since "12am" refers to "midnight."